client/dashboard/dashboard-charts/block.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@woocommerce/components';
+import ReportChart from 'analytics/components/report-chart';
+import './block.scss';
+class ChartBlock extends Component {
```

Comment:
Should we make a dashboard block/chart component available under the components package or is that not useful yet? I can't recall how extendable we are making the dashboard for v1, but this seems like something reusable with the `endpoint` prop for example.